[2024-02-19T18:29:57.197+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v08.create_postgres_table scheduled__2024-02-13T00:00:00+00:00 [queued]>
[2024-02-19T18:29:57.208+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v08.create_postgres_table scheduled__2024-02-13T00:00:00+00:00 [queued]>
[2024-02-19T18:29:57.208+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-02-19T18:29:57.238+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2024-02-13 00:00:00+00:00
[2024-02-19T18:29:57.274+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v08', 'create_postgres_table', 'scheduled__2024-02-13T00:00:00+00:00', '--job-id', '448', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp2wy7b3yq']
[2024-02-19T18:29:57.282+0000] {standard_task_runner.py:88} INFO - Job 448: Subtask create_postgres_table
[2024-02-19T18:29:57.249+0000] {standard_task_runner.py:60} INFO - Started process 1159 to run task
[2024-02-19T18:29:57.395+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_with_postgres_operator_v08.create_postgres_table scheduled__2024-02-13T00:00:00+00:00 [running]> on host 55ac849b3b96
[2024-02-19T18:29:57.636+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Shiv' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v08' AIRFLOW_CTX_TASK_ID='create_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2024-02-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-13T00:00:00+00:00'
[2024-02-19T18:29:57.637+0000] {sql.py:276} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS dag_runs (
                dt DATE,
                dag_id VARCHAR,
                PRIMARY KEY (dt, dag_id)
            );
        
[2024-02-19T18:29:57.658+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-02-19T18:29:57.675+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-02-19T18:29:57.693+0000] {sql.py:450} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS dag_runs (
                dt DATE,
                dag_id VARCHAR,
                PRIMARY KEY (dt, dag_id)
            );
        , parameters: None
[2024-02-19T18:29:57.773+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v08, task_id=create_postgres_table, execution_date=20240213T000000, start_date=20240219T182957, end_date=20240219T182957
[2024-02-19T18:29:57.917+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-19T18:29:58.037+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
